src/main/scala/group/scala/karazin/mongo4s/model.scala

```diff
@@ -318,10 +318,11 @@ object model:
     final case class SortByCount[SortByCount]($sortByCount: SortByCount)
 
     object UnionWith:
-      type Command[Pipeline] = UnionWithCommand[Pipeline] | String
-      final case class UnionWithCommand[Pipeline](coll: String, pipeline: Pipeline)
+      type Command = UnionWithCommand | String
+      /** WARN!!! `pipeline` cannot include the $out and $merge stages */
```

Comment:
Could you explain, please?

Reply:
In documentation it's mentioned that there shouldn't be $out and $merge stages in UnionWith.
I've put it here as a reminder (because our Pipeline includes those commands, and one has to be wary about that).